src/components/ListTasks.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { FlatList } from 'react-native';
+import Task from './Task';
+
+const data = [
+  { title: 'task1', description: 'description of task1', id: '1' },
+  { title: 'task2', description: 'description of task2', id: '2' },
+  { title: 'task3', description: 'description of task3', id: '3' },
+];
+
+class ListTasks extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tasks: data,
+    };
+  }
+
+  render() {
+    const { tasks } = this.state;
+    return (
+      // eslint-disable-next-line react/jsx-filename-extension
+      <FlatList
+        data={tasks}
+        renderItem={({ item }) => (
```

Comment:
I suggest declaring this function outside the JSX and calling it's reference.

Apart from making your JSX more legible, this have some performance [issues](https://medium.freecodecamp.org/why-arrow-functions-and-bind-in-reacts-render-are-problematic-f1c08b060e36).